.github/workflows/build_pull_request.yml

```
@@ -62,76 +38,27 @@ jobs:
       - name: Set up Flutter
         uses: subosito/flutter-action@v2
         with:
-          channel: ${{ inputs.flutter-channel }}
-          flutter-version: ${{ inputs.flutter-version }}
+          channel: stable
```

Allow Flutter cache on PR (this shouldn't affect us much)

```suggestion
          channel: stable
          cache: true
```